client/javascript/fixtures/risk-assessment-questions.json

```diff
@@ -69,7 +69,7 @@
         "no": { "text": "No" }
       }
     },
-    "description": "<p class='lede text'>Examples of evidence to look for:</p> <ul class='list list-bullet text u-margin-bottom-charlie'><li>they've previously been a victim of prison violence</li><li>they are in debt to another prisoner</li><li>there's reasons why someone in this prison might want to hurt them</li></ul>",
+    "description": "<p class='lede text'>Examples of evidence to look for:</p> <ul class='list list-bullet text'><li>they've previously been a victim of prison violence</li><li>they are in debt to another prisoner</li><li>there's reasons why someone in this prison might want to hurt them</li></ul>",
```

Comment:
Do the changes in this file belong in this Jira ticket?

Reply:
No it's an earlier commit `CSRA-593: Remove margin from bullet points`